src/builtin/math.rs

```diff
@@ -207,14 +206,12 @@ fn min(args: CallArgs, parser: &mut Parser<'_>) -> SassResult<Value> {
     let mut min = nums.next().unwrap();
 
     for num in nums {
-        if Value::Dimension(num.0.clone(), num.1.clone())
-            .cmp(
-                Value::Dimension(min.0.clone(), min.1.clone()),
-                Op::LessThan,
-                span,
+        if ValueVisitor::new(parser, span)
+            .less_than(
+                HigherIntermediateValue::Literal(Value::Dimension(num.0.clone(), num.1.clone())),
+                HigherIntermediateValue::Literal(Value::Dimension(min.0.clone(), min.1.clone())),
```

Comment:
Is our numbers `Copy` or does it require clone?

Reply:
`Number`s are clone only as they *could* be the arbitrary precision, heap allocated `Big` variant. The `.clone`s for the small variant (up to 19 digits) I believe are the same as a `Copy`, so there shouldn't be any performance penalty here.